src/azul/plugins/repository/tdr_anvil/__init__.py

```diff
@@ -147,6 +150,11 @@ def optional_key_ref_to_entity_ref(key_ref: Optional[KeyReference]) -> Optional[
             for link in links
         )
 
+    def _validate_drs_uri(self, drs_uri: Optional[str]) -> None:
+        if drs_uri is not None:
+            drs_uri = DRSCompactPseudoURI.maybe_parse(drs_uri)
+            require(drs_uri.scheme == 'drs')
```

Comment:
All DRS URIs start with `'drs:/'`. This should be checked  in the `parse` factory method.